infra/terraform/modules/network/proxy_private.tf

```diff
@@ -0,0 +1,85 @@
+################### proxy A ######################
```

Comment:
Add that this is about subnets ? i.e `private_subnets` or something like that

Reply:
private_subnets is in the title of the file. This is just to identify where start the private_subnet_a (for proxy) and the private_subnet_b (for proxy).